roles/nsupdate_web/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 - name: Including major version specific variables.
   include_vars: "{{ item }}"
   with_first_found:
-    - "{{ ansible_distribution | lower }}_{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution | lower | replace(' ', '_') }}_{{ ansible_distribution_major_version }}.yml"
```

Comment:
What is the `ansible_distribution` var for Leap 42.3?  Just trying to understand what this is for.

Reply:
@djgalloway "openSUSE Leap"

Reply:
Ah, okay.  Thanks.